docs/structures/examples.md

```diff
@@ -0,0 +1,132 @@
+
+* **descriptive text only**: these aren't really examples, but enable authors to describe or link to examples if for any reason they don't want to include the example source itself.
+
+* **static code samples**: these consist of blocks of source code, optionally with descriptive text. The source code may not be fully functional (for example, the example might require so much setup or other dependencies that it's not practical to make a functional example).
+
+* **live samples**: these consist of blocks of source code, optionally with descriptive text, and a specification for the size of an iframe in which the result can be presented. The code must be functional: so a consumer can package it into a document and have it run in an iframe.
+
+Note that this doesn't include interactive examples, which are handled separately.
+
+## Authoring format
+
+To include examples for an item (e.g. the `<video>` element), set the `examples` key in the item's meta.yaml file. This key is specified as a list of paths. These might be relative to meta.yaml or absolute, in which case it must begin from the "/content" directory.
+
+For example:
+
+```yaml
+examples:
+    - examples/simple-example
+    - examples/multiple-sources-example
+```
+
+```yaml
+examples:
+    - /content/html/elements/shared-examples/some-example
+```
+
+### Example syntax
+
+Inside each directory listed under the `examples` property are up to four files, all optional:
+
+* description.md
+* example.css
+* example.html
+* example.js
+
+#### description.md
+
+This is a Markdown file containing some front matter. The front matter may contain up to three properties:
+
+* `title`: Optional. A human-readable title for this example. If present this will be represented in the JSON and used as a section heading by the MDN renderer. If an item contains more than one example with a title, then all titles must be unique. If any examples for an item have titles, then all of them must (so it's OK for there to be multiple examples with no titles, but not for some to have titles and others not).
+
+* `height`: Optional, but if given, then `width` must also be given. If this is present then the example is treated as a live sample, and this represents the height of the result iframe in pixels.
+
+* `width`: Optional, but if given, then `height` must also be given. If this is present then the example is treated as a live sample, and this represents the width of the result iframe in pixels.
+
+The rest of "description.md" consists of a description of this example.
+
+Example "description.md":
+
+```
+---
+title: Marking up an abbreviation semantically
+width: 672
+height: 192
+---
+To mark up an abbreviation without providing an expansion or
+description, simply use `<abbr>` without any attributes, as seen in this example.
+```
+
+#### example.css, example.html, example.js
+
+These files contain the example's CSS, HTML, and JS. Only one file of each type may be included.
+
```

Comment:
It could mention that they're all optional. Right?
But what if none of them are present?

Reply:
> It could mention that they're all optional. Right?

That's already mentioned just above: https://github.com/wbamberg/stumptown-experiment/blob/document-structures/docs/structures/examples.md#example-syntax

>  But what if none of them are present?

If it's a live example, that's an error. We could say that.